lib/lich/gameobj.rb

```diff
@@ -7,7 +7,7 @@ def method_missing(*); end
 
 # Tell GameObj where to find the GameObj data XML file
 class GameObj
-  DATA_DIR = Pathname.new('./scripts').expand_path
+  DATA_DIR = Pathname.new('./dist').expand_path
```

Comment:
We can probably the existing XML and the generated XML side by side by either setting different values in different specs or more explicitly reloading the data. One thing to keep in mind is that `GameObj` memoizes this load, so we would need to be careful to ensure we are testing the right file if they are both used. I didn't do anything to clear this state between my examples because I thought it would be slow to reload the XML file all the time, but we could probably mitigate that impact.